cpp/lib/include/opendnp3/gen/EventMode.h

```diff
@@ -48,7 +48,8 @@ enum class EventMode : uint8_t
   Force = 0x1,
   /// Never produce an event regardless of changes
   Suppress = 0x2,
-  /// Send an event directly to the event buffer, bypassing the static value completely
+  /// Force the creation of an event bypassing detection mechanism, but does not
```

Comment:
We need to update this comment via the code generator. It will also update java/.net

Reply:
🤦‍♂️ My bad